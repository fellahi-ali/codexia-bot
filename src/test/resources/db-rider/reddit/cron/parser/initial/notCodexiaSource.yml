github_repo:
    -
        id: 1
        uuid: "groovy: UUID.randomUUID().toString()"
        external_id: 202275988
        full_name: test1/test1
        html_url: https://github.com/test1/test1
        repo_created_at: 2019-08-14 04:42:36

github_repo_source:
    -
        uuid: "groovy: UUID.randomUUID().toString()"
        github_repo_id: 1
        source: "HACKERNEWS"
        external_id: "2222"
