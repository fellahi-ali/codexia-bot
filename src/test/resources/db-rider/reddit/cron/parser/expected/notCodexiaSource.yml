github_repo_source:
    -
        github_repo_id: 1
        source: "HACKERNEWS"
        external_id: "2222"
